build/teamcity/cockroach/ci/tests/roachtest_stress_impl.sh

```diff
@@ -10,7 +10,11 @@ if [[ ! -f ~/.ssh/id_rsa.pub ]]; then
   ssh-keygen -q -C "roachtest-stress $(date)" -N "" -f ~/.ssh/id_rsa
 fi
 
-source "$root/build/teamcity/cockroach/nightlies/roachtest_compile_bits.sh"
+arch=amd64
+if [[ ${FIPS_ENABLED:-0} == 1 ]]; then
+  arch=amd64-fips
+fi
```

Comment:
AFAIK, this build is only triggered manually. Consider whether the `arch` could be passed directly as an environment variable via the TC UI (allowing us to stress tests on ARM too, for example).